myuw-ios/WebViewController.swift

```diff
@@ -40,20 +40,6 @@ class WebViewController: UIViewController, WKNavigationDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // MARK: - Notification Center
```

Comment:
Not sure what the NotificationCenter gives us... since all it watches is the AppDelegate's willEnterForegroundNotification. Seemed redundant.

Reply:
Sure, we can add it back in if we ever do need it.